### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by thiagomurakami**
#### What problem is this solving?

Documentationn

#### Checklist/Reminders

- [x] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️ | Technical improvements

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,86 @@
...
+This app is used by [`checkout-step-group`](https://github.com/vtex-apps/checkout-step-group) and shouldn't be used individually. Should you find the need to use it, you need to:
+
+1. Add it as a dependency in your `manifest.json`:
+```json
+{
+  "dependencies": {
+    "vtex.checkout-shipping": "0.x"
+  }
+}
+```
```
Comment (lucasecdb) [inline docs/README.md]: we could recommend just running `vtex add vtex.checkout-shipping` (I recently learned this command 😄)

File: docs/README.md [hunk 2]
```diff
@@ -1 +1,86 @@
...
+2. Import the components in your code:
+
+```jsx
+import React from 'react'
+import { ShippingPreview, ShippingForm } from 'vtex.checkout-shipping'
+
+const Form = ({ isPreviewMode = false }) => {
+  if (isPreviewMode) {
+    return (
+      <ShippingPreview />
```
Comment (lucasecdb) [inline docs/README.md]: the component is `ShippingSummary`, `ShippingPreview` is another monster 😂
```suggestion
import { ShippingSummary, ShippingForm } from 'vtex.checkout-shipping'

const Form = ({ isPreviewMode = false }) => {
  if (isPreviewMode) {
    return (
      <ShippingSummary />
```

File: docs/README.md [hunk 3]
```diff
@@ -1 +1,86 @@
...
+
+  return (
+    // no need to pass in any props, as the component
+    // uses all information provided by the `order-manager` app.
+    <ShippingForm />
+  )
+}
+```
+
+Note that you need to have `OrderFormProvider` from `vtex.order-manager` app somewhere above this
```
Comment (lucasecdb) [inline docs/README.md]: can we change this to `vtex.order-shipping`? Order manager is just a transient dependency

**Review by kevinch — DISMISSED** [review]

File: docs/README.md [hunk 4]
```diff
@@ -1 +1,79 @@
...
+
+  return (
+    // no need to pass in any props, as the component
+    // uses all information provided by the `order-manager` app.
+    <ShippingForm />
+  )
+}
+```
+
+Note that you need to have `OrderFormProvider` from `vtex.order-shipping` app somewhere above this
```
Comment (lucasecdb) [inline docs/README.md]:
```suggestion
Note that you need to have `OrderShippingProvider` from `vtex.order-shipping` app somewhere above this
```

**Review by lucasecdb — DISMISSED** [review]
LGTM, fix then ship it 😄

**Review by lucasecdb — APPROVED** [review]

**Review by darlenedms — APPROVED** [review]

**Pull request merged**